runtime/wasmer.go

```diff
@@ -90,6 +95,16 @@ func (r *Runtime) Stop() {
 	r.vm.Close()
 }
 
+func (r *Runtime) StorageRoot() (common.Hash, error) {
+	if r == nil {
```

Comment:
I think this first check should be unnecessary. Checking the trie is nil could be done in `NewRuntime` once, so we don't have to do it every time here.

Reply:
ok removed, and moved the trie check to NewRuntime